boards/px4/fmu-v4pro/src/init.c

```diff
@@ -110,7 +110,7 @@ __EXPORT void board_peripheral_reset(int ms)
 {
 	/* set the peripheral and sensor rails off */
 	stm32_gpiowrite(GPIO_VDD_3V3_PERIPH_EN, 0);
-	stm32_gpiowrite(GPIO_VDD_3V3_SENSORS_EN, 0);
+	board_control_spi_sensors_power(false, 0xffff);
```

Comment:
Are we ever not using a busmask of `0xffff`?

Reply:
in V5x

Reply:
> in V5x

I don't believe so. https://github.com/PX4/Firmware/pull/14221/files#diff-8e2fa2c853114fc9e393de59a7442b16R108